packages/edge-gateway/test/index.spec.js

```diff
@@ -52,7 +52,7 @@ test('Gets content', async (t) => {
   const { mf } = t.context
 
   const response = await mf.dispatchFetch(
-    'https://bafkreihl44bu5rqxctfvl3ahcln7gnjgmjqi7v5wfwojqwriqnq7wo4n7u.ipfs.localhost:8787'
+    'https://bafkreihl44bu5rqxctfvl3ahcln7gnjgmjqi7v5wfwojqwriqnq7wo4n7u.ipfs.localhost:8787?foo=test'
```

Comment:
would be cool to add a test here specifically for search params. Then adding a mock in smoke https://github.com/web3-storage/reads/tree/main/packages/edge-gateway/test/mocks/ipfs.io to handle a different response there :)

Reply:
@vasco-santos I've added a new test here.  I'm unfamiliar with the codebase, so I'm still not exactly sure why the fixtures being added to the container seem to take precedence over the mocks managed by smoke (i.e., the `fetch` will return the fixture if it finds one and the smoke function never gets invoked) but it seemed to finally work by adding a new cid only to the smoke file (not the fixture).  🤷 

Let me know if there's anything you'd like changed.

Reply:
```
so I'm still not exactly sure why the fixtures being added to the container seem to take precedence over the mocks managed by smoke
```

The tests are setup as a request race of between a IPFS node running inside a docker container, and two other mock servers. Mock serves in this case should be faster and return asap. So, I guess in this case we need to add a new mock file with a different more specific name. See https://github.com/sinedied/smoke#query-parameters

Reply:
Updated.  I've removed the else blocks in the mocks with the query param checks and added the new more specific query param mock files.